docs/source/conf.py

```diff
@@ -14,7 +14,7 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath('../..'))
```

Comment:
this is not a reliable way of doing this.

1. consider getting the abspath of `__file__`, and walk up two directory tree components.
2. (preferable) use pkg_resources to find the root.

Reply:
Fair. I think this conf.py file is only called by the Makefile, so due to the limited use patterns it's not as fragile as you think. However, can't hurt to make it more robust! 

I wasn't able to figure out how `pkg_resources` improves things, since it sends us to `starfish/starfish/` which is also wrong (we need `starfish/`). Anchored this to `__file__`.